binaries/coordinator/examples/graphs/python_test.yml

```diff
@@ -0,0 +1,29 @@
+communication:
+  zenoh:
+    prefix: /foo
+
+nodes:
+  - id: static-string
+    custom:
+      run: python examples/nodes/python/static_string.py
+      outputs:
+        - string
+
+  - id: python-printer
+    custom:
+      run: python examples/nodes/python/printer.py
+      inputs:
+          time: static-string/string
```

Comment:
The input name `time` does not really make sense here. Maybe something like `string` or `hello` instead?